simtools/simtel/simtel_runner_camera_efficiency.py

```diff
@@ -102,6 +104,8 @@ def _make_run_command(self, **kwargs):  # pylint: disable=unused-argument
             )
 
         command = str(self._simtel_source_path.joinpath("sim_telarray/bin/testeff"))
+        if self.nsb_spectrum is not None:
+            command += f" -fnsb {self.nsb_spectrum}"
```

Comment:
just wondering, how did testeff handle it before, when there was no nsb file? does it have a default file somewhere?